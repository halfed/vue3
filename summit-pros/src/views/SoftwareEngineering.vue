<template>
    <div>
        <h1>Custom Software Engineering</h1>
    </div>
</template>