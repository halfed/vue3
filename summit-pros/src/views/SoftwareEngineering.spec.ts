import { mount } from '@vue/test-utils';
import { describe, it, expect } from 'vitest';
import SoftwareEngineering from './SoftwareEngineering.vue';

describe('SoftwareEngineering', () => {
     it('renders correctly', () => {
        const wrapper = mount(SoftwareEngineering);
        expect(wrapper.element).toMatchSnapshot();
      });
});