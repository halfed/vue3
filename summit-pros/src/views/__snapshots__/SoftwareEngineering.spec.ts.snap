// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SoftwareEngineering > renders correctly 1`] = `
<div
  class="flat-page"
>
  <div>
    <img
      class="page-image"
      src="/src/assets/software-engineering.jpg"
    />
  </div>
  <h1>
    Automotive Repair & Diagnostics
  </h1>
  <p>
     Our software engineering team partners with startups, SMBs, and enterprise clients to design, build, and maintain robust digital solutions. We specialize in practical, scalable development that supports real business operations not just code for the sake of it. 
  </p>
  <h2>
    Capabilities include:
  </h2>
  <ul>
    <li>
      Full-stack web and mobile application development
    </li>
    <li>
      Internal business tools and automation systems
    </li>
    <li>
      API integration and custom dashboards
    </li>
    <li>
      Cloud infrastructure consulting (AWS, Azure, GCP)
    </li>
    <li>
      Agile project management and DevOps support
    </li>
    <li>
      Ongoing maintenance, updates, and scalability planning
    </li>
  </ul>
  <p>
     Whether you need to streamline your business operations, build a customer-facing app, or replace outdated systems, MTEK delivers secure, scalable software tailored to your exact needs on time and on budget. 
  </p>
</div>
`;
